Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -12,2045 +12,2318 @@ namespace FluentAssertions.Equivalency.Specs;
 
 public class SelectionRulesSpecs
 {
-    private enum LocalOtherType : byte
+    public class Basic
     {
-        Default,
-        NonDefault
-    }
-
-    private enum LocalType : byte
-    {
-        Default,
-        NonDefault
-    }
-
-    [Fact]
-    public void When_specific_properties_have_been_specified_it_should_ignore_the_other_properties()
-    {
-        // Arrange
-        var subject = new
+        [Fact]
+        public void When_a_property_is_an_indexer_it_should_be_ignored()
         {
-            Age = 36,
-            Birthdate = new DateTime(1973, 9, 20),
-            Name = "John"
-        };
+            // Arrange
+            ClassWithIndexer expected = new()
```

Comment:
Is it intentional that you switch from `var foo = new SomeType()` to `SomeType foo = new()`?
As far as I can tell we currently only use taget-type new expressions for local variables five places.

_If_ we want to switch completely to the newly supported syntax, I think we should:
* do it in a separate PR, 
* across the whole repo and
* enforce it via `.editorconfig`.


Using different syntax makes it difficult for contributors to stay consistent.

Reply:
Undid the changes. Will look at that in a follow-up PR.